### PR TITLE
Use only position command interface for Beta driver

### PR DESCRIPTION
**Pull request by livanov93**
This PR removes all velocity command interface stuff on actively developed branch as a result of milestones for Beta driver release.

### CODE REVIEW

File: ur_robot_driver/src/hardware_interface.cpp [hunk 1]
```diff
@@ -374,35 +376,26 @@ return_type URPositionHardwareInterface::write()
...
-
-    // create a lambda substract functor
-    std::function<double(double, double)> substractor = [](double a, double b) { return std::abs(a - b); };
-
-    // create a position difference vector
-    std::vector<double> pos_diff;
-    pos_diff.resize(urcl_position_commands_.size());
-    std::transform(urcl_position_commands_.begin(), urcl_position_commands_.end(), urcl_position_commands_old_.begin(),
-                   pos_diff.begin(), substractor);
+    if (!position_interface_in_use_)
```
Comment (destogl) [inline ur_robot_driver/src/hardware_interface.cpp:379]: With this logic here you will not call `writeJointCommand` in the same update call when you detect `position_interface_in_use_ = true;`

Is this correct? The old velocity implementation will do this in the same loop.

**Review by destogl — CHANGES_REQUESTED** [review]
Did you intend to remove velocity control completely? Why not control this completely using `positio_interface_in_use_` flag?

The code is clean and everything is fine, but I have too little information about motivation to do this like it is done.

File: ur_robot_driver/src/hardware_interface.cpp (hunk 1 above)
Comment (livanov93) [reply ur_robot_driver/src/hardware_interface.cpp:379]: Correct!
https://github.com/PickNikRobotics/Universal_Robots_ROS2_Driver/pull/32/commits/4aad52397470f02486bd9bc269fbb368361dad00

**Comment by livanov93** [comment]
> Did you intend to remove velocity control completely? Why not control this completely using `positio_interface_in_use_` flag?
> 
> The code is clean and everything is fine, but I have too little information about motivation to do this like it is done.

Yes, decided to fully remove it for Beta driver milestone. Also to avoid any misunderstandings for the driver users for now if they take a look at the code (commenst like there is a method writeJointCommand in velocity mode, but the driver does not support it...).

**Review by destogl — APPROVED** [review]

**Pull request merged**